test/unit/spec/MfaVerify_spec.js

```diff
@@ -4035,7 +4035,7 @@ function (Okta,
 
```

Comment:
test above checks for `test.form.isTOTP()`, should we change that to `test.form.isHOTP()`?

Reply:
The actual form used is totp form. That is why I used this method.

Reply:
ah right! in verify we use the same form as totp 👍